rust/models/src/host_info.rs

```diff
@@ -24,8 +24,8 @@ pub struct HostInfo {
     pub finished: u32,
     #[cfg_attr(
         feature = "serde_support",
-        serde(skip_serializing_if = "Vec::is_empty")
+        serde(skip_serializing_if = "Option::is_none")
     )]
     /// IPs of hosts, that are currently scanned.
-    pub scanning: Vec<String>,
+    pub scanning: Option<Vec<String>>,
```

Comment:
why do you need `Option<Vec<...>>`? Over just leave it empty?

Reply:
this as given some errors before the in file storage was merged, because serde deserializer.

Reply:
is that still the case?

Reply:
yes it is. Smoketest fails if when getting the status and there is no scanning member